export/build

```diff
@@ -93,21 +93,24 @@ def make_header(header):
         return text
 
     # Other keywords are silently discarded
-    keywords = ["Description", "Time", "Memory", "Usage"]
+    keywords = ["Description", "Time", "Memory", "Usage", "Name"]
 
     l = []
+    heading = "\\ttheader{%s}" % processed_fn
     for h in keywords:
-        if h in header:
-            if h == "Usage":
-                # TODO: quite a hack
-                l.append("{\\leftskip=\\usagedimen\\item{\\bf Usage:}\\raggedright\\simpleverb|%s\\egroup\\par}" % highlight_and_texify(header[h]))
-            else:
-                l.append("{\\bf %s:} %s" % (h, texify(header[h])))
-
-    if not l:
-        return ""
-    else:
-        return "\n\n".join(l)
+        if h not in header:
+            continue
+
+        if h == "Usage":
+            # TODO: quite a hack
+            l.append("{\\leftskip=\\usagedimen\\item{\\bf Usage:}\\raggedright\\simpleverb|%s\\egroup\\par}" % highlight_and_texify(header[h]))
```

Comment:
If I'm not mistaken, we can use Python's raw string literals to get rid of the double escapes:
```python
r"{\leftskip}"
```

Reply:
Great, thanks!